packages/api/src/posthog.ts

```diff
@@ -0,0 +1,11 @@
+import { PostHog } from "posthog-node";
+
+import { env } from "@blobscan/env";
+
+let posthog: PostHog | null = null;
+
+if (env.POSTHOG_KEY && env.POSTHOG_HOST) {
```

Comment:
Let's rename POSTHOG_KEY to POSTHOG_ID

Reply:
Done [8fe0a01](https://github.com/Blobscan/blobscan/pull/578/commits/8fe0a011357a382d6a1fb92e85101aa8019f65c0)